content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md

```diff
@@ -0,0 +1,29 @@
+
+HPE Ezmeral Unified Analytics is a hybrid and multi-cloud solution. This is a fancy way of saying that, at the time of GA (general availability), HPE Ezmeral Unified Analytics will run everywhere you want it to run. It can be installed on-premises, at the edge, or in a public cloud. We will provide our next-gen HPE Ezmeral Unified Analytics product in an appliance-like form factor that can be installed and ready to use in under 10 minutes. The built-in global namespace will allow you to easily connect multiple unified analytics workloads and data fabrics to share compute and data across different clouds and different workflows.
+
+**Let your opinion be known!**
+
+One could consider HPE Ezmeral Unified Analytics as an opinionated set of software. Part of the purpose of this beta is to ensure that our opinion makes sense. We are trying to select the most important open source products in data engineering, data analytics, and data science. Because we don’t expect our customers to choose all of our tools, we designed our workflows and pipelines to allow easy replacement of an included component by an alternative. We try to provide a comprehensive selection of products, but we also try to stay out of your way if you have preferences. For example, while we include Super Set for building reports and dashboards, if you prefer to use Tableau, our JDBC Presto API’s are available for a quick swap. If you prefer using Weights and Bias’, you can swap out our ML Flow registry solution. Models can be published to Kserve or Sagemaker Endpoints.
+
+As part of the beta program, you will have access to an HPE Ezmeral Unified Analytics environment hosted on HPE infrastructure. We will provide you with initial training to get you up to speed on the various open source frameworks that are included. We will also provide you a Slack channel for support questions. We will rely on you to provide feedback so we can change direction, if necessary. This initial beta test will be available for about 3 months and will then move to a stage where you will be able to deploy HPE Ezmeral Unified Analytics in your own environment for additional testing and feedback. During and immediately after the initial beta period, we will request feedback informally and through surveys.
```

Comment:
Slack Channel....do we know the link for the slack channel? Once we know it, we could add it as hotlink.

Reply:
I don't have the link for it yet, but yes, I put the words in here so we could link it once we knew it. Unfortunately, I'm not sure how to add links in the editing piece.